flow/probes/ebpf.go

```diff
@@ -53,22 +50,16 @@ const (
 	ebpfUpdate = 2 * time.Second
 )
 
-// EBPFFlow describes the userland side of an eBPF flow
-type EBPFFlow struct {
-	start time.Time
-	last  time.Time
-	lastK int64
-}
-
 // EBPFProbe the eBPF probe
 type EBPFProbe struct {
-	probeNodeTID string
-	fd           int
-	flowTable    *flow.Table
-	module       *elf.Module
-	fmap         *elf.Map
-	expire       time.Duration
-	quit         chan bool
+	probeNodeTID   string
+	fd             int
+	flowTable      *flow.Table
+	module         *elf.Module
+	fmap           *elf.Map
+	expire         time.Duration
+	quit           chan bool
+	flow_pool_size int
```

Comment:
keep the camelCase style flowPoolSize
Sorry, my fault, on my last comment ;)

Reply:
done